syncd/syncd_saiswitch.cpp

```diff
@@ -100,21 +100,19 @@ std::string SaiSwitch::saiGetHardwareInfo() const
     return std::string(info);
 }
 
-std::vector<sai_object_id_t> SaiSwitch::saiGetPortList() const
+void SaiSwitch::saiInitPortList()
```

Comment:
saiGetinitPortList

Reply:
also shit method should be const, and return list of ports, if you want to store them inside SaiSwitch object, call this method inside SaiSwitch constructor and then populate m_port_rid_list, and change name to something like m_init_port_list_rids etc

ps. why you need init port list? we agreed thta port number will not change during runtime for example ports will not be splitted/merged